plugin-packages/multimedia/src/video/video-component.ts

```
@@ -105,12 +105,14 @@ export class VideoComponent extends BaseRenderComponent {
     } = options;
 
     this.transparent = transparent;
-    if (video) {
-      this.video = (video as unknown as Asset<HTMLVideoElement>).data;
+    if (video && video.id) {
+      const videoAsset = this.engine.findObject<Asset<HTMLVideoElement>>(video);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)